ts/store/actions/navigationHistory.ts

```diff
@@ -13,8 +13,14 @@ export const navigationHistoryPop = createStandardAction(
   "NAVIGATION_HISTORY_POP"
 )();
 
+export const navigationHistoryMultiplePop = createStandardAction(
```

Comment:
Can't we just have a single pop action with a option parameter that default to 1?

Reply:
@laurapugliese

Reply:
@fpersico I make the change

Reply:
@fpersico done in commit 4c2d0e4 ;)